website/server/libs/payments.js

```diff
@@ -125,13 +130,39 @@ api.addSubToGroupUser = async function addSubToGroupUser (member, group) {
         {name: 'EMAIL', content: getUserInfo(member, ['email']).email},
         {name: 'PAYMENT_METHOD', content: memberPlan.paymentMethod},
         {name: 'PURCHASED_PLAN', content: JSON.stringify(memberPlan)},
-        {name: 'ACTION_NEEDED', content: 'User has joined group plan. Tell them to cancel subscription then give them free sub.'},
+        {name: 'ACTION_NEEDED', content: 'User has joined group plan and has been told to cancel their subscription then email us. Ensure they do that then give them free sub.'},
+        // TODO User won't get email instructions if they've opted out of all emails. See if we can make this email an exception and if not, report here whether they've opted out.
       ]);
     }
 
-    if ((ignorePaymentPlan || ignoreCustomerId) && !customerHasCancelledGroupPlan) return;
+    if ((ignorePaymentPlan || ignoreCustomerId) && !customerHasCancelledGroupPlan) {
+      // member has been added to group plan but their subscription will not be changed
+      // automatically so they need a special message in the email
+      if (memberPlan.paymentMethod === this.constants.GOOGLE_PAYMENT_METHOD) {
+        previousSubscriptionType = 'Google';
```

Comment:
That's correct, `previousSubscriptionType` is solely to pass a specific string that Mandrill will recognise into the email content. But I do see your point about constants. I've added new ones specifically for this so it won't matter if we change the existing constants. I've also made the values that are passed to Mandrill more complex so there's less chance we'll change them by accident when changing other constants. I've pushed a commit for that.

I've adjusted the Mandrill template to match. It's not in use yet with production code so this change won't affect current emails.